common/component_interface_specs/include/component_interface_specs/vehicle.hpp

```diff
@@ -71,6 +74,27 @@ struct EnergyStatus
   static constexpr auto durability = RMW_QOS_POLICY_DURABILITY_VOLATILE;
 };
 
+struct DoorCommand
+{
+  using Service = autoware_adapi_v1_msgs::srv::SetDoorCommand;
+  static constexpr char name[] = "/vehicle/doors/command";
+};
+
+struct DoorLayout
+{
+  using Service = autoware_adapi_v1_msgs::srv::GetDoorLayout;
+  static constexpr char name[] = "/vehicle/doors/layout";
+};
+
+struct DoorStatus
+{
+  using Message = autoware_adapi_v1_msgs::msg::DoorStatusArray;
+  static constexpr char name[] = "/vehicle/doors/status";
```

Comment:
Should we make the names consistent with other command/status messages?

* /vehicle/command/doors_command
* /vehicle/status/doors_status
* /vehicle/doors_layout

or should we change the current message for control commands to something like 
`/vehicle/control/actuation_command` ?

Reply:
@yukkysaito What do you think?

Reply:
@mitsudome-r @yukkysaito 
When thinking of it as a vehicle interface, it feels strange to have separate namespaces for commands and status into `/vehicle` and `/control`. So I propose to change the vehicle interface as follows. 

| current topic name                     | new topic name                          |
| -------------------------------------- | --------------------------------------- |
| /vehicle/status/control_mode           | /vehicle/control/control_mode           |
| /vehicle/status/gear_status            | /vehicle/control/gear_status            |
| /vehicle/status/turn_indicators_status | /vehicle/control/turn_indicators_status |
| /vehicle/status/velocity_status        | /vehicle/control/velocity_status        |
| /vehicle/status/steering_status        | /vehicle/control/steering_status        |
| /control/command/control_cmd           | /vehicle/control/control_cmd            |
| /control/command/gear_cmd              | /vehicle/control/gear_cmd               |
| /control/command/turn_indicators_cmd   | /vehicle/control/turn_indicators_cmd    |
| -                                      | /vehicle/doors/layout                   |
| -                                      | /vehicle/doors/status                   |
| -                                      | /vehicle/doors/command                  |